ADA/git_databricks.qmd

```diff
@@ -0,0 +1,221 @@
+---
+  title: "Databricks and version control"
+---
+
+<p class="text-muted">
+
+Guidance for analysts on how to connect Databricks to Github and DevOps
+
+</p>
+
+------------------------------------------------------------------------
+
+## Why should I use git with Databricks?
+
+------------------------------------------------------------------------
+
+#### Easier collaboration
+
+------------------------------------------------------------------------
+
+When you're working on notebooks in Databricks without a git connection, they tend to be saved in your own personal Workspace. This means that they are only accessible by you, unless you share them. This has the potential to cause issues if you're on leave and someone needs to run any code you have stored in a notebook within Databricks. If the notebooks are stored in a Github or DevOps repo, your team can all access them by cloning the repo in Databricks.
+
+\
+
+
+------------------------------------------------------------------------
+
+#### Version control
+
+------------------------------------------------------------------------
+
+Databricks autosaves your notebooks as you're working on them, making version control more difficult. If you use git, you'll be able to see the full version history of your work and easily roll back to older versions if you need to.
+
+\
+
+------------------------------------------------------------------------
+
+## Github or DevOps?
+
+------------------------------------------------------------------------
+
+Either Github or DevOps repos can be used in connection with Databricks, but we advise you to follow the guidance relating to public and private repos in our [What is git for?](https://dfe-analytical-services.github.io/analysts-guide/learning-development/git.html#what-is-git-for) section.
```

Comment:
It's probably me that's started this, but "GitHub or DevOps" sounds weird because GitHub effectively is just an implementation of DevOps (i.e. I think it's GitHub DevOps and Azure DevOps). So probably better to say "GitHub or Azure DevOps" here (I got a very weird look once from someone with actual technical knowledge when I said a sentence along the lines of "we use both GitHub and DevOps").

Reply:
I certainly said DevOps a lot on this page 🤣